String_Indexing_Operations.py

```diff
@@ -0,0 +1,10 @@
+print('Enter a string value')
```

Comment:
I don't understand what this program is doing. Please rework and make changes according to the instructions given.

<img width="325" alt="image" src="https://user-images.githubusercontent.com/38807024/200740289-d3d7046b-efb4-47b3-be1c-2003e69a2bfb.png">